js/front/dataset/index.js

```diff
@@ -122,7 +122,11 @@ new Vue({
             location.hash = `resource${communityPrefix}-${id}`;
             const modal = this.$modal(
                 ResourceModal,
-                {datasetId: this.dataset['@id'], resource: this.resourceFromJsonLd(resourceJsonLd)}
+                {
+                    datasetId: this.dataset['@id'],
+                    resource: this.resourceFromJsonLd(resourceJsonLd),
+                    isCommunity: isCommunity,
```

Comment:
Isn't it possible to read this boolean from the `resource` paramter attributes ?

Reply:
Probably yes 👍

Reply:
Nope, nothing in there.

Reply:
OK, so LGTM
(maybe we should think about exposing this attribute in resources)